independent-projects/resteasy-reactive/common/processor/src/main/java/org/jboss/resteasy/reactive/common/processor/scanning/ResteasyReactiveScanner.java

```diff
@@ -37,17 +37,27 @@
 public class ResteasyReactiveScanner {
 
     public static final Map<DotName, String> BUILTIN_HTTP_ANNOTATIONS_TO_METHOD;
+    public static final Map<String, DotName> METHOD_TO_BUILTIN_HTTP_ANNOTATIONS;
```

Comment:
Where is this used?

Reply:
Nevermind, found it in https://github.com/quarkusio/quarkus/pull/20195/files#diff-4e34a8d1bf348000ad6478d0e6152cc5728dd01bce175e3ed38461f80eba1d8cR182